config/Config.py

```diff
@@ -1,37 +1,48 @@
 import os
+from selenium import webdriver
+
+options = webdriver.ChromeOptions()
 
 common_caps = {
-    "browserstack.user" : "BROWSERSTACK_USERNAME",
-    "browserstack.key" : "BROWSERSTACK_ACCESS_KEY",
-    "build" : "browserstack-build-1",
-    "browserstack.debug" : "true"
+    "userName" : "BROWSERSTACK_USERNAME",
+    "accessKey" : "BROWSERSTACK_ACCESS_KEY",
+    "buildName" : "browserstack-build-1",
+    "debug" : "true"
 }
 
+browsers = ["Chrome", "Safari", "Chrome"]
+browser_versions = ["80", "latest", "latest"]
+
 envs = [{
     "os" : "Windows",
-    "os_version" : "10",
-    "browser" : "Chrome",
-    "browser_version" : "latest"
+    "osVersion" : "10",
 },
 {
     "os" : "OS X",
-    "os_version" : "Big Sur",
-    "browser" : "Safari",
-    "browser_version" : "latest"
+    "osVersion" : "Big Sur",
 },
 {
-    "device" : "Samsung Galaxy S22",
-    "os_version" : "12"
+    "deviceName" : "Samsung Galaxy S22",
+    "osVersion" : "12"
```

Comment:
We don't need to have browsers and browser_versions as separate arrays, can be placed inside envs only - having it this way makes it a bit more complicated - also we don't need browser_version in mobile sessions... let's keep browser latest for both..

Reply:
Addressed via commit 7b3f7c37b8cfe6b2e2d8925f2d3878efb38a2377